src/plugins/stake-plugins/types.ts

```diff
@@ -160,8 +160,28 @@ export interface StakePosition {
 // Stake Plugin
 // -----------------------------------------------------------------------------
 
+export interface StakePolicyFilter {
+  wallet?: EdgeCurrencyWallet
+  currencyCode?: string
```

Comment:
I don't like seeing `currencyCode` used for any money stuff. It needs to become a purely a display thing going forward, not something we can compare based on.

However, I see that currencyCode exists in a bunch of places here, so maybe it's too hard to fix. If it really is a hard fix, I guess we can leave it.

Reply:
That's a good point. I looked at the potential work involved and currencyCode is pretty pervasive throughout the staking plugin so I'd rather punt